generate_golden.sh

```diff
@@ -68,11 +68,11 @@ scaffold_test_project() {
 		$kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
 		$kb create webhook --group crew --version v1 --kind FirstMate --conversion
 		$kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
-		# TODO(droot): Adding a second group is a valid test case and kubebuilder is expected to report an error in this case. It
-		# doesn't do that currently so leaving it commented so that we can enable it later.
-		# $kb create api --group ship --version v1beta1 --kind Frigate --example=false --controller=true --resource=true --make=false
-		$kb create api --group core --version v1 --kind Namespace --example=false --controller=true --resource=false --namespaced=false --make=false
-		# $kb create api --group policy --version v1beta1 --kind HealthCheckPolicy --example=false --controller=true --resource=true --namespaced=false --make=false
+
+		header_text 'Switching to multigroup layout ...'
```

Comment:
It seems the generated single-group and multi-group code are mixed in the same project `testdata/project-v2`.
I'd suggest keep them separate:
- single group one still in `testdata/project-v2`
- multi-group one probably lives in `testdata/project-v2-multi-group`

The reason behind this is that we sometimes point our users to the testdata projects as examples, so we'd better keep them clean.

Reply:
I agree. Tks